syncer/syncer.go

```diff
@@ -1087,6 +1087,7 @@ func (s *Syncer) Run(ctx context.Context) (err error) {
 				return err2
 			}
 
+			currentLocation = shardingReSync.currLocation.Clone()
```

Comment:
Is `nextLocation.Clone()` better in here?

Reply:
update in 9f48852